content/questions/foundations/question-107.md

```diff
@@ -0,0 +1,17 @@
+---
+archetype: "questions"
+title: "Question 107"
+question: "Which permissions can setup billing or assign billing managers for their organization(s)? (Select 2)"
+draft: false
+---
+
+
+> https://learn.microsoft.com/en-us/training/modules/github-introduction-administration/2-what-is-github-administration
```

Comment:
Maybe [this](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/adding-a-billing-manager-to-your-organization) is a better source? Also seems like existing Billing Managers can also invite additional people to become billing managers

Reply:
Gotcha, I was going off the study guides that GitHub resources linked here: https://resources.github.com/learn/certifications.  But it is good to cross reference then official GitHub instead.